src/packet-saprfc.c

```diff
@@ -485,8 +498,51 @@ dissect_saprfc_tables_compressed(tvbuff_t *tvb, packet_info *pinfo, proto_tree *
 			tvb_set_child_real_data_tvbuff(tvb, next_tvb);
 			add_new_data_source(pinfo, next_tvb, "Uncompressed Table Data");
 
+			structure = proto_tree_add_item(tree, hf_saprfc_table_structure, structure_tvb, 4, structure_length, ENC_NA);
+			structure_tree = proto_item_add_subtree(structure, ett_saprfc);
+
+			/* Parse row structure */
+			field_count = tvb_get_guint8(structure_tvb, structure_offset); structure_offset+=1;
+			field_types = alloca(field_count);
+			field_lengths = alloca(field_count);
+			field_offsets = alloca(field_count * sizeof(guint32));
```

Comment:
We should be using wmem's allocators here, check https://gitlab.com/wireshark/wireshark/-/blob/master/doc/README.wmem. I think the packet scope would be probably fine for this use (e.g. `wmem_alloc(wmem_packet_scope(), ..)`).